test/augmentation/test_augmentation_3d.py

```diff
@@ -24,979 +24,979 @@
 )
 
 
-# class TestRandomHorizontalFlip3D:
```

Comment:
@shijianjian but what's the problem with this tests ?

Reply:
No problems on those. Only a typo actually. I commented other tests to play with the new tests.